pyproject.toml

```diff
@@ -93,6 +93,8 @@ datasets = [
     "pyvista>=0.34.2",
     # radiant-mlhub 0.3+ required for newer tqdm support required by lightning
     "radiant-mlhub>=0.3",
+    # source-cooperative 
+    "cooperative>=0.1.12",
```

Comment:
https://pypi.org/project/cooperative/ isn't related to what you're trying to do

Reply:
Source Cooperative uses azcopy for downloading; maybe [azure-storage-blob](https://pypi.org/project/azure-storage-blob/) will be useful.